clients/desktop/src/vault/qr/upload/UploadQrPage/UploadQrPageWithoutVault.tsx

```
@@ -1,20 +1,19 @@
-import { UploadQrView } from './UploadQrView'
+import { Match } from '../../../../lib/ui/base/Match'
+import { useAppNavigate } from '../../../../navigation/hooks/useAppNavigate'
+import { useAppPathParams } from '../../../../navigation/hooks/useAppPathParams'
+import { useNavigateBack } from '../../../../navigation/hooks/useNavigationBack'
+import { FlowPageHeader } from '../../../../ui/flow/FlowPageHeader'
+import { ScanQrView } from '../ScanQrView'
+import { UploadQrView } from '../UploadQrView'
+import { StyledPageContent } from './UploadQRPage.styled'
 
 const uploadQrViews = ['scan', 'upload'] as const
 type UploadQrView = (typeof uploadQrViews)[number]
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UploadQrView'. Consider to delete it or rename it.

'UploadQrView' is defined here:


(lint/suspicious/noRedeclare)